pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -1087,8 +1088,15 @@ func traceNextConfigReq(ctx *zedagentContext) bool {
 	if !isNettraceEnabled(ctx) {
```

Comment:
Looks like this and the corresponding info function only differ in the last*NetdumpPub they check and the functions are getting more complicated. Would it make sense to introduce a common function?

Reply:
Makes sense - done